README.md

```diff
@@ -31,13 +29,253 @@ _NB: There is no compatibility with Windows environments at this time._
 6. _(Optional)_ To run the UI with its own API, first copy `.env-ui.sample` as `.env-ui`. Please see the [configuration](#configuration) section for more details.
 7. _(Optional)_ Run `make run-ui` (or `make run-ui-arm` on Apple Silicon) to have the Web UI available on <http://localhost:8088> (in production mode). Its HTTP auth credentials are set in `.env-ui`. The UI API also has a frontend for API documentation (default <http://localhost:3002>).
 
-> If you want to run the UI app in development mode, i.e. with HMR enabled, please follow the steps in the [Development (UI)](#development-ui) section.
+#### Docker Guide Requirements
+
+- Unix-based operating system (e.g. Debian, Arch, Mac OS)
+- [Docker Engine](https://docs.docker.com/engine/) `1.27+`
+- Makefile toolchain `GNU Make 3.81`
+
+ℹ️ _**NOTE:** There is no compatibility with Windows environments at this time._
+
+To help expedite a lot of the docker commands, a lot has been abstracted using make commands with the `Makefile`.
+Included in each command are the docker commands to show what is being run.
+
+#### Create Docker Configuration Files
+
+Make a copy of the following environment variables files:
+
+```bash
+# FROM: ./
+
+cp .env-api.sample .env-api;
+cp .env-issuer.sample .env-issuer;
+# (Optional - For Issuer UI)
+cp .env-ui.sample .env-ui;
+```
+
+#### Node Issuer Configuration
+
+The `.env-issuer` will be loaded into the [docker compose initiliaser](/infrastructure/local/docker-compose.yml)
+
+You can use one of the following RPC providers:
+
+- [Chainstack](https://chainstack.com/)
+- [Ankr](https://ankr.com/)
+- [QuickNode](https://quicknode.com/)
+- [Alchemy](https://www.alchemy.com/)
+- [Infura](https://www.infura.io/)
+
+If you want to get up and running with a free public forwarding url, look at [Getting A Public URL](#getting-a-public-url).
+
+**File:** `./.env-issuer`
+
+```bash
+# ...
+
+ISSUER_SERVER_URL=https://unique-forwaring-or-public-url.ngrok-free.app
+# Defaults for Basic Auth in Base64 ("user-issuer:password-issuer" = "dXNlci1pc3N1ZXI6cGFzc3dvcmQtaXNzdWVy")
+# If you just want to get started, don't change these
+ISSUER_API_AUTH_USER=user-issuer
+ISSUER_API_AUTH_PASSWORD=password-issuer
+# !!!MUST BE SET or other steps will not work
+ISSUER_ETHEREUM_URL=<YOUR_RPC_PROVIDER_URI_ENDPOINT>
+```
+
+ℹ️ **NOTE:** In case you have loaded the vault multiple times and want a fresh start, run the following to remove remnant data:
+
+```bash
+# FROM: ./
+
+rm ./infrastructure/local/.vault/data/init.out;
```

Comment:
Why are there semi-colons after these commands?

Reply:
This is to make copying and pasting the code easier so that if they copied the entire code and there was formatting issues that it would still execute properly with the bash `;` putting an end to each command.